meetings/2025-09-24.md

```diff
@@ -0,0 +1,211 @@
+* SC: If there is a disagreement on that expectation, it should be put in writing. If a detailed clarification is needed, a proposal can be put forwarded to the CG.
+
+### Proposal to fix invalid moves and archiving
+
+Proposed by SC.
+
+* SC: from https://github.com/solid-contrib/shapes/issues/5#issuecomment-3326730800 :
+  >I propose that the repositories be moved back to their original location, unarchived, and any settings changes undone. The CG as a whole can then decide what needs to be managed and coordinate with ODI.
+* SC: Please unarchive repositories that were archived without CG's consent, e.g., https://github.com/solid/test-suite-panel/
+* Jesse: I can provide context. We at the ODI are going through a cleanup of github.org/solid to do a better job of going through issue backlogs. We posted a spreadsheet open for comment for over two weeks and only enacted prescribed actions on things that were not commented on. Sarven then requested things be moved back. I'd like to discuss whether some work items can be considered not active and moved to solid-contrib.
+* eP: The least friction approach, I really appreciate the work to cleanup and groom GitHub. It is unfortunate Sarven didn't manage to be here today. solid-contrib is a bit unclear whether it is under ODI perview. We could archive things.
+* Jesse: We will moderate solid-contrib and add the code of conduct. We're not saying we won't ever touch a repo there, it's just that we're not actively monitoring that space. I'd like to check whether archiving is ok. We need a CG consensus to confirm the CG is ok with archiving.
+* Michiel: I value the work Jesse is doing here and we should let him move and archive as makes sense. Whatever is clearer to readers.
+* Jesse: I looked at the issue and Sarven asked to unarchive.
+* Michal: I didn't understand what Sarven said. I'm not sure if solid and solid-contrib belongs to CG or ODI?
+* Jesse: ODI has legal ownership of solid GitHub org and responsibility to monitor those repos. The only one deffered to the CG is solid/specification
+* eP: I found the quote I looked for:
+    * SC: If there is a proposal to archive repos: shapes, test-suite-panel, authentication-panel, solid-wg-charter, I have no objection. However, do not move them to another org or rename. 
+* eP: Just to avoid the debate we can archive things, instead of moving.
+* Jesse: For archiving, I am comfortable moving things back and archiving.
+
+### Ownership Question
+
+* Michal: I don't understand the meaning of legal ownership (especially for solid-contrib and practitioners things).
+* Jesse: We are moderating all repository on the solid and solid-contrib GitHub org and all codebases are MIT license.
+* Michiel: We can think of it as stewardship of these repos rather than ownership.
```

Comment:
That was my view as well. And that is also why ODI does not have authority on moving things, archiving. It has the necessary privileges to admin the space but as hosts, CG material is decided by the CG.